sdk/communication/communication-chat/package.json

```diff
@@ -8,7 +8,7 @@
   "types": "types/communication-chat.d.ts",
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:autorest": "autorest ./swagger/README.md --typescript --version=3.0.6267 --v3 --disable-async-iterators=true --package-version=1.0.0-beta.5 && rushx format",
```

Comment:
There's a version number in line 3

Reply:
This was already bumped in a previous PR https://github.com/Azure/azure-sdk-for-js/pull/14216